gap/PGMatrix.gd

```diff
@@ -1,93 +1,203 @@
 #!
 #!
-#! @Chapter Hyperbolic lattice symmetries
+#! @Chapter Representations of point groups
 
+
+#! @Section Point-group matrices
+#!
+#! Point-group matrices are representations of group elements in the point group $G\cong\Delta/\Gamma$. 
+#! These matrices can efficiently be constructed. Given any symmetry `g` in the point group $G$ and a
+#! set `I` of all translation generators $\gamma_i$ of the translation group $\Gamma$, a corresponding
+#! point-group matrix can be determined through set of the symmetry transformed elements $g \gamma_i g^{-1}$.
+#! Point-group matrices enable the conduction of symmetry analysis on any given (supercell) model graph.
+#!
+#! @Section PGMatrixElements
+#! @SectionLabel PGMatrixElements
+#!
+#!   - <Ref Oper='Signature' Label='for PGMatrixElements' />:
```

Comment:
Having looked at the function `PGMatrix` it seems like we might need to think about whether to keep `PGMatrix` as is for efficiency reasons and have some code duplicity in the (new) `EvaluateRGMatrixRep` function or call `EvaluateRGMatrixRep` internally in `PGMatrix`. The latter would be more elegant, but if we need to reconstruct object several times, it might not be good.